CHANGELOG.md

```diff
@@ -291,6 +291,11 @@ at velocity 0 it would look the same as its tail (but you can't have 0 velocity)
 - Added compressors to synths, kits, audio clips, and kit rows. The compressor can be enabled and edited from their respective menus.
 - Compressor behavior has been changed to reduce clipping. Songs made with community release 1.0.x may need to have their volume manually adjusted to compensate. This is done via affect entire in song/arranger mode, or by entering the `SONG MENU` by pressing `SELECT` in Song View and navigating to  `MASTER > VOLUME`
 
+#### <ins>Polyphony / Voice Count </ins>
```

Comment:
The changelog should probably have a much shorter entry - if this file gets too long it will stop being useful